python/heterocl/rptdisp.py

```diff
@@ -0,0 +1,338 @@
+import pandas as pd
+from tabulate import tabulate
+
+class RptDisp(object):
```

Comment:
If merged with `report.py`, we can rename it to be `Displayer` to make it clearer.